rootfs/scheduler/k8s.py

```diff
@@ -145,8 +153,8 @@ def _get_old_rc(self, name, app_type):
                 break
         if exists:
             return prev_rc
-        else:
```

Comment:
The way I've always approached it (can't remember if it is in K&R or similar style guides) is that you are already returning a default value at the end, no need to `else` those, `exists` is the special condition, etc etc :)